README.md

```diff
@@ -4,60 +4,583 @@ A test-framework-agnostic set of helpers for testing Ember.js applications.
 
 ## Usage
 
+The exports of this library are intended to be utility functions that can be used to bring the
+standard Ember testing experience to any testing framework.
+
 This library is best used by way of a test-framework-specific
 wrapper, such as [ember-qunit](https://github.com/emberjs/ember-qunit) or
 [ember-mocha](https://github.com/emberjs/ember-mocha).
 
-## Test Helpers
+## API
 
-The exports of this library are intended to be utility functions that can be used to bring the
-standard Ember testing experience to any testing framework.
+<!-- Generated by documentation.js. Update this documentation by updating the source code. -->
+
+#### Table of Contents
+
+-   [DOM Interaction Helpers](#dom-interaction-helpers)
+    -   [click](#click)
+    -   [tap](#tap)
+    -   [focus](#focus)
+    -   [blur](#blur)
+    -   [triggerEvent](#triggerevent)
+    -   [triggerKeyEvent](#triggerkeyevent)
+    -   [fillIn](#fillin)
+-   [Routing Helpers](#routing-helpers)
+    -   [visit](#visit)
+    -   [currentRouteName](#currentroutename)
+    -   [currentURL](#currenturl)
+-   [Rendering Helpers](#rendering-helpers)
+    -   [render](#render)
+    -   [clearRender](#clearrender)
+-   [Wait Helpers](#wait-helpers)
+    -   [waitFor](#waitfor)
+    -   [waitUntil](#waituntil)
+    -   [settled](#settled)
+    -   [isSettled](#issettled)
+    -   [getSettledState](#getsettledstate)
+-   [Pause Helpers](#pause-helpers)
+    -   [pauseTest](#pausetest)
+    -   [resumeTest](#resumetest)
+-   [Test Framework APIs](#test-framework-apis)
+    -   [setResolver](#setresolver)
+    -   [getResolver](#getresolver)
+    -   [setupContext](#setupcontext)
+    -   [getContext](#getcontext)
+    -   [setContext](#setcontext)
+    -   [unsetContext](#unsetcontext)
+    -   [teardownContext](#teardowncontext)
+    -   [setupRenderingContext](#setuprenderingcontext)
+    -   [teardownRenderingContext](#teardownrenderingcontext)
+    -   [setApplication](#setapplication)
+    -   [setupApplicationContext](#setupapplicationcontext)
+    -   [teardownApplicationContext](#teardownapplicationcontext)
+    -   [validateErrorHandler](#validateerrorhandler)
+
+### DOM Interaction Helpers
+
+
+
+
+#### click
+
+Clicks on the specified target.
+
+Sends a number of events intending to simulate a "real" user clicking on an
+element.
+
+For non-focusable elements the following events are triggered (in order):
+
+-   `mousedown`
+-   `mouseup`
+-   `click`
+
+For focusable (e.g. form control) elements the following events are triggered
+(in order):
+
+-   `mousedown`
+-   `focus`
+-   `focusin`
+-   `mouseup`
+-   `click`
+
+The exact listing of events that are triggered may change over time as needed
+to continue to emulate how actual browsers handle clicking a given element.
+
+**Parameters**
```

Comment:
On mobile this renders the same as the h4 of the functions which makes it somewhat hard to read and find stuff 🤔

Reply:
Yeah, I think that is also true on desktop.